src/Compilers/CSharp/Portable/Symbols/Source/SourceMethodSymbolWithAttributes.cs

```diff
@@ -1039,11 +1039,13 @@ private void DecodeInterceptsLocationChecksumBased(DecodeWellKnownAttributeArgum
             switch (referencedToken)
             {
                 case { Parent: SimpleNameSyntax { Parent: MemberAccessExpressionSyntax { Parent: InvocationExpressionSyntax } memberAccess } rhs } when memberAccess.Name == rhs:
```

Comment:
>case { Parent: SimpleNameSyntax { Parent: MemberAccessExpressionSyntax [](http://example.com/codeflow?start=16&length=70)

Is this case redundant given the `{ Parent: SimpleNameSyntax { Parent: MemberAccessExpressionSyntax memberAccess } rhs } when memberAccess.Name == rhs` case below?

Reply:
No, the case below is for an error recovery path (the MemberAccess's parent is not an Invocation).